pkgs/games/maptool/default.nix

```diff
@@ -13,15 +13,10 @@
 let
   pname = "maptool";
   version = "1.13.2";
-  repoBase = "https://github.com/RPTools/${pname}";
+  repoBase = "https://github.com/RPTools/maptool";
```

Comment:
```suggestion
```

Reply:
Nope, this deduplicates the base repository URL between the source and the update script.

Reply:
You can get this url via src.url. We do not need to cripple the URL.

Reply:
Are you aware that the value of `src.url` is `https://github.com/RPTools/maptool/releases/download/1.13.2/maptool-1.13.2-x86_64.pkg.tar.zst`?

Are you proposing splitting on `/` and selecting the first n components, or stripping the last n? Because that seems way less transparent than defining a variable for the URL base. I don't see any problem with this approach.

Reply:
I don't see how ‘You can get this url via src.url’ could possibly mean that, but... come on, is this a blocker? Does being the maintainer of this package grant me the leeway to take the oh-so-bold position that deduplication is good actually, because in the future I want it to be obvious if the source and the update script URLs drift apart?

Run a `grep -rF 'url = "${'` on Nixpkgs if you think I'm being some sort of maverick by using this pattern. It's not exactly uncommon. Criticisms of functionality are more than welcome, and violations of the Nixpkgs house coding style also make good feedback, but this is something else altogether and I beg one of you to just click the dang merge button on this months-old PR if this is the only nit left to pick.

Reply:
Definitely not a blocker for me. I'll give Sandro 24 hours to react before I merge it, so ping me next night if it's still not merged.